### DISCUSSION TITLE
1.1-a Desktop 3D Case link is 404?

### DISCUSSION THREAD

**Discussion by spalmen** [erichelgeson/BlueSCSI-v1#263] [Q&A] [answered]
Hi. The v1.1 internal mounting bracket STL.zip link is 404, at least for me.

I'm accessing it via the v1 wiki:
https://github.com/erichelgeson/BlueSCSI/wiki

From there, selecting '3D Cases'
https://github.com/erichelgeson/BlueSCSI/wiki/3D-Cases

That page lists 1.0 and 1.1 desktop brackets. 

The 1.0-a/b/c links to:
https://github.com/erichelgeson/BlueSCSI/tree/main/3D-Files/1.0
but the 1.0 STL wireframe doesn't look like it will fit my 1.1 board.

1.1-a Desktop section displays a link to:
https://github.com/erichelgeson/BlueSCSI/blob/main/3D-Files/1.1/bluescsi11a_kerosmacmodsv2.stl.zip
but that gives me a error:
404 - page not found
The main branch of BlueSCSI does not contain the path 3D-Files/1.1/bluescsi11a_kerosmacmodsv2.stl.zip.

What am doing wrong? (I tried this with Mac Firefox and Chrome, just in case it's a Safari thing.) Thank you. Cheers…

**Comment by spalmen** [comment]
Oh, and if I manually change the 1.0 URL to:
https://github.com/erichelgeson/BlueSCSI/tree/main/3D-Files/1.1
there I see 'external-db25/' which has STLs for the external board. I'm looking for the v1.1 internal mount.

**Comment by erichelgeson** [comment, marked as answer]
I see someone moved them around and it accidentally got deleted - I've added it back under https://github.com/erichelgeson/BlueSCSI/blob/main/3D-Files/1.1/internal-50pin/bluescsi11a_kerosmacmodsv2.stl.zip and updated the wiki. Thanks for seeing something and saying something. Enjoy!
